### ISSUE TITLE
Move website to new top level git repo

### ISSUE THREAD

**Issue by lilleswing** [deepchem/deepchem#1055]
We want to do more custom work on our website.  
Our website is currently static html and javascript with no server based component.

By separating the website to its own github repo we can more clearly show what is auto-generated and what is html code.

We will also get more time via travis to do deploys to the website which we are not doing right now.

**Comment by lilleswing** [comment]
It would still be in the deepchem organization

https://github.com/deepchem/deepchem_website

**Comment by rbharath** [comment]
Yes, I think this would be a good idea. We could do interesting things with the website as we move past the pure static html/javascript model.

**Comment by lilleswing** [comment]
The MIME type of "text/css" is being lost on _static/bootstrap_sphinx.css accross the proxy we have.  This is causing bootstrap_sphinx.css to be read as a text file and not loading in the @import on its first line.
